FirebaseAuth/Sources/Swift/Backend/RPC/SignUpNewUserRequest.swift

```diff
@@ -77,6 +82,12 @@ class SignUpNewUserRequest: IdentityToolkitRequest, AuthRPCRequest {
       @brief The password inputed by the user.
    */
   private(set) var displayName: String?
+
+  /** @property idToken
+      @brief he idToken of the user.
```

Comment:
typo: `he` instead of `The`

Reply:
Thanks. Looks like I shouldn't include the dash in the commit message to skip CI https://docs.github.com/en/actions/managing-workflow-runs/skipping-workflow-runs